Source/Core/Core/Boot/Boot.cpp

```diff
@@ -351,39 +351,44 @@ bool CBoot::BootUp()
       DVDInterface::SetVolumeName(_StartupPara.m_strDefaultISO);
     }
 
-    if (!EmulatedBS2(dolWii))
-    {
-      // Set up MSR and the BAT SPR registers.
-      UReg_MSR& m_MSR = ((UReg_MSR&)PowerPC::ppcState.msr);
-      m_MSR.FP = 1;
-      m_MSR.DR = 1;
-      m_MSR.IR = 1;
-      m_MSR.EE = 1;
-      PowerPC::ppcState.spr[SPR_IBAT0U] = 0x80001fff;
-      PowerPC::ppcState.spr[SPR_IBAT0L] = 0x00000002;
-      PowerPC::ppcState.spr[SPR_IBAT4U] = 0x90001fff;
-      PowerPC::ppcState.spr[SPR_IBAT4L] = 0x10000002;
-      PowerPC::ppcState.spr[SPR_DBAT0U] = 0x80001fff;
-      PowerPC::ppcState.spr[SPR_DBAT0L] = 0x00000002;
-      PowerPC::ppcState.spr[SPR_DBAT1U] = 0xc0001fff;
-      PowerPC::ppcState.spr[SPR_DBAT1L] = 0x0000002a;
-      PowerPC::ppcState.spr[SPR_DBAT4U] = 0x90001fff;
-      PowerPC::ppcState.spr[SPR_DBAT4L] = 0x10000002;
-      PowerPC::ppcState.spr[SPR_DBAT5U] = 0xd0001fff;
-      PowerPC::ppcState.spr[SPR_DBAT5L] = 0x1000002a;
-      if (dolLoader.IsWii())
-        HID4.SBE = 1;
-      PowerPC::DBATUpdated();
-      PowerPC::IBATUpdated();
+    // Set up MSR and the BAT SPR registers.
+    UReg_MSR& m_MSR = ((UReg_MSR&)PowerPC::ppcState.msr);
+    m_MSR.FP = 1;
+    m_MSR.DR = 1;
+    m_MSR.IR = 1;
+    m_MSR.EE = 1;
+    PowerPC::ppcState.spr[SPR_IBAT0U] = 0x80001fff;
+    PowerPC::ppcState.spr[SPR_IBAT0L] = 0x00000002;
+    PowerPC::ppcState.spr[SPR_IBAT4U] = 0x90001fff;
+    PowerPC::ppcState.spr[SPR_IBAT4L] = 0x10000002;
+    PowerPC::ppcState.spr[SPR_DBAT0U] = 0x80001fff;
+    PowerPC::ppcState.spr[SPR_DBAT0L] = 0x00000002;
+    PowerPC::ppcState.spr[SPR_DBAT1U] = 0xc0001fff;
+    PowerPC::ppcState.spr[SPR_DBAT1L] = 0x0000002a;
+    PowerPC::ppcState.spr[SPR_DBAT4U] = 0x90001fff;
+    PowerPC::ppcState.spr[SPR_DBAT4L] = 0x10000002;
+    PowerPC::ppcState.spr[SPR_DBAT5U] = 0xd0001fff;
+    PowerPC::ppcState.spr[SPR_DBAT5L] = 0x1000002a;
+    if (dolWii)
+      HID4.SBE = 1;
```

Comment:
Good question. `SetupSPR_Wii`, `SetupBAT_Wii`, `SetupPPCState_Wii` or something along those lines? `_Wii` suffix since GC does something similar; except not everything due to the Wii having more SPRs available.